static/_app/tools/tradecraft/index.html

```
@@ -0,0 +1,21 @@
+
+<head>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+<title>Tradecraft</title>
+<link rel="shortcut icon" href="/boltlogo.png">
+<link rel="canonical" href="https://slope-game.github.io/tradecraft.html">
+<meta 
```

HTMLHint (1.5.0)

[error] The value of attribute [ name ] must be in double quotes.

(attr-value-double-quotes)